src/App.js

```diff
@@ -1,24 +1,24 @@
-import logo from './logo.svg';
+
 import './App.css';
+import SearchBar from './components/SearchBar';
+
+
+
+
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+    <header>
+      <h1>Search Recipes!</h1>
```

Comment:
I love the chalkboard look for your site and "Search Recipes!" is a perfect, clear header that provides immediate instruction for the user